### DISCUSSION TITLE
已部署应用修改参数重建后显示找不到容器

### DISCUSSION THREAD

**Discussion by dididaguai** [1Panel-dev/1Panel#6316] [Q&A]
### 1Panel 版本

v1.10.14-lts

### 问题描述

例如ONLYOFFICE Docs这个应用部署后，修改参数，会提示重建后生效，点击重建后会显示异常，查看异常提示找不到xxx容器。
在容器界面看，容器名称在原有基础上又加了一串字符串+下划线。实际服务正常运行。

### 重现步骤

例如ONLYOFFICE Docs这个应用部署后，修改参数，会提示重建后生效，点击重建后会显示异常，查看异常提示找不到xxx容器。
在容器界面看，容器名称在原有基础上又加了一串字符串+下划线。实际服务正常运行。

**Comment by wanghe-fit2cloud** [comment]
通过应用商店创建的应用，容器名称的命名规则是：`1Panel-应用名称-随机串`，如图所示：
> 修改参数等操作不会改变应用名称等信息。我们在本地环境中也未复现你提到的问题。
<img width="1029" alt="image" src="https://github.com/user-attachments/assets/03316d72-7db0-4d8c-b53a-ac8d1b2e6320">

**Comment by dididaguai** [comment]
我知道什么问题了，修改后会提示重建应用，实际是会自动重建不用手动，我之前那错误是点击确定后手动重建，然后应该是冲突了导致的，比如容器名称变为随机串-随机串-应用名称（创建时手动修改为固定应用名称了的）
